### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by OskarStark**
Please review commit by commit

### CODE REVIEW

**Comment by OskarStark** [comment]
Are you fine with adding PHP-CS-Fixer?

**Comment by Chrico** [comment]
Thanks a lot for your PR @OskarStark . Sadly travis is now failing with a lot phpcs warnings/errors. Would you be so kind to fix them before we're going to merge your pull? :)

File: src/Formatter/LogzIoFormatter.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function __construct(int $batchMode = self::BATCH_MODE_NEWLINES, bool $ap
      *
      * @param array $record
      *
-     * @see https://support.logz.io/hc/en-us/articles/210206885-How-can-I-get-Logz-io-to-read-the-timestamp-within-a-JSON-log-
+     * @link https://support.logz.io/hc/en-us/articles/210206885-How-can-I-get-Logz-io-to-read-the-timestamp-within-a-JSON-log-
```
Comment (bueltge) [inline src/Formatter/LogzIoFormatter.php]: Why? Please can you explain why is that better.

Comment (OskarStark) [reply src/Formatter/LogzIoFormatter.php]: "better" is the wrong word here, its mostly used for internal references between methods and *can* be used for links:
```php
  * @see MyClass::$items           For the property whose items are counted.
  * @see MyClass::setItems()       To set the items for this collection.
```

but `@link` is explicitely only for external [references](http://docs.phpdoc.org/references/phpdoc/tags/link.html)

Comment (bueltge) [reply src/Formatter/LogzIoFormatter.php]: Thanks for clarify!

Comment (OskarStark) [reply src/Formatter/LogzIoFormatter.php]: Welcome 😊👍🏻

File: composer.json [hunk 2]
```diff
@@ -17,13 +17,13 @@
     }
   ],
   "require": {
-    "php": ">=7.0",
-    "monolog/monolog": "^1.23",
-    "ext-curl": "*"
+    "php": "^7.0",
+    "ext-curl": "*",
+    "ext-json": "*",
```
Comment (OskarStark) [inline composer.json]: Hmm IIRC, json_encode is only used in test, so we should move it to require-dev

Comment (bueltge) [reply composer.json]: +1

**Comment by OskarStark** [comment]
> Would you be so kind to fix them before we're going to merge your pull? :)

Sadly I cannot fix it, lets take this for example:

_LogzIoHandler.php_
```php
    protected function write(array $record)
    {
        $this->send($record['formatted']);
    }
```

this is the warning (line 65):
```
  65 | ERROR   | [x] Parenthesis content must be padded by a space (NeutronStandard.Whitespace.RequireParenthesisSpacing.Missing)
  65 | ERROR   | [x] Parenthesis content must be padded by a space (NeutronStandard.Whitespace.RequireParenthesisSpacing.Missing)
```

quite clear so I assume changing to this should fix it:
```diff
-        $this->send($record['formatted']);
+        $this->send( $record['formatted'] );
```

afterwards I get this error:
```
  65 | ERROR   | [x] Space after opening parenthesis of function call prohibited (PSR2.Methods.FunctionCallSignature.SpaceAfterOpenBracket)
  65 | ERROR   | [x] Expected 0 spaces before closing bracket; 1 found (PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket)
```

Can you help?

**Comment by OskarStark** [comment]
Any feedback?

**Comment by bueltge** [comment]
The lead dev on this project is currently in holidays, after them I will talk with him.

**Comment by Chrico** [comment]
Howdy @OskarStark and happy new year!

Sorry for the slow feedback. I've checked your pull request out and tested locally what goes wrong here. It seems this is related to an outdated version of our coding standard. Via composer the version 0.8.0 is pulled, while we're already on version 0.13.x. If you update the version, most of those `Parenthesis`-errors are gone. :-)

Could you please change in `require-dev` to following `"inpsyde/php-coding-standards": "~0.13"` ?

**Comment by OskarStark** [comment]
Done, but 1 left:
```
FILE: /Users/oskar.stark/dev/misc/logzio-monolog/src/Handler/LogzIoHandler.php
------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------
 69 | WARNING | Argument type is missing
    |         | (Inpsyde.CodeQuality.ArgumentTypeDeclaration.NoArgumentType)
------------------------------------------------------------------------------------------------------
```

is about this method:
```php
    protected function send($data)
    {
        $handle = curl_init();

        curl_setopt($handle, CURLOPT_URL, $this->endpoint);
        curl_setopt($handle, CURLOPT_POST, true);
        curl_setopt($handle, CURLOPT_POSTFIELDS, $data);
        curl_setopt($handle, CURLOPT_HTTPHEADER, ['Content-Type: application/json']);
        curl_setopt($handle, CURLOPT_RETURNTRANSFER, true);

        Util::execute($handle);
    }
```

but the method ```handleBatch()``` is returning `mixed`, so I cannot type hint the argument for `send($data)`

How to proceed?

**Comment by Chrico** [comment]
Howdy!

Nice work, thanks so far. :-) We could add a `// phpcs:disable Inpsyde.CodeQuality.ArgumentTypeDeclaration.NoArgumentType` above the `protected function`. But for me it's ok. This is failing because the method is kind of restricted by the implementation of Monolog.

**Comment by OskarStark** [comment]
Will add the comment

**Pull request merged**